docs/operating-scylla/procedures/cluster-management/add-node-to-cluster.rst

```diff
@@ -5,49 +5,73 @@ Adding a New Node Into an Existing ScyllaDB Cluster (Out Scale)
 When you add a new node, other nodes in the cluster stream data to the new node. This operation is called bootstrapping and may
```

Comment:
@kbr- is this still true (bootstapping may take hours?)

Reply:
Of course, data needs to be moved around like before.

(btw tagged wrong account)